### ISSUE TITLE
Deploy using shiro DB for live users  

### ISSUE THREAD

**Issue by Zarquan** [wfau/aglais#758]
Test the system for deploying live user accounts using Shiro-DB authentication.
* Check both Dave and Stelios can access the shiro-hash data on our data node.
* Check login works for existing users (nch, dcr, stv, zrq).
* Document the process for creating new live users.

**Comment by Zarquan** [comment]
The `import-live-users` function combines the public data from `/deployments/common/users/live-users.yml` and the private password hashes from our data node. I've run several tests using this code in the ansible client container to add the live users:
```
source /deployments/zeppelin/bin/create-user-tools.sh

import-live-users

list-shiro-info \
    /tmp/live-users.json

list-ceph-info \
    /tmp/live-users.json
```

**Comment by Zarquan** [comment]
@stvoutsin can you confirm this works for you too.

**Comment by Zarquan** [comment]
Fixed in #888, but GitHub won't allow us to link more than 10 issues to a PR.

**Comment by Zarquan** [comment]
Fixed in #888
